README.rst

```diff
@@ -9,10 +9,10 @@ Using the ``async``/``await`` capabilities of Python3,
 
 .. code:: python3
 
-   >>> from usim import each, run
+   >>> from usim import delay, run
    >>>
    >>> async def metronome(period: float, sound: str):
-   ...     async for now in each(delay=period):
+   ...     async for now in delay(value=period):
```

Comment:
I recommend not using a keyword argument here - at least to me it reads better to have ``... in delay(period)``.

```suggestion
   ...     async for now in delay(period):
```

Alternatively, consider renaming the argument to ``of`` or ``by`` - ``of`` would also fit nicely for ``interval(of=period)``.

Reply:
Removed the keyword. The other options looked strange.